bin/main

```diff
@@ -228,7 +228,7 @@ def _save_artifact(
     resp_create = json.load(urllib.request.urlopen(req_create))
 
     req_upload = urllib.request.Request(
-        f'{resp_create["fileContainerResourceUrl"]}?itemPath={artifact_name}/data.json',  # noqa: E501
+        f'{resp_create['fileContainerResourceUrl']}?itemPath={artifact_name}/data.json',  # noqa: E501
```

Comment:
Bad bot :robot: 

This is causing `bin/main:231:26: E999 SyntaxError: f-string: unmatched '['` error

Reply:
the bot's not wrong, just the pre-commit config is outdated (using a version that only supports 3.12+)